src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -11,18 +11,26 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
 
+	private String _feedbackToUser;
+	
+    public final String getfeedbackToUser(){
```

Comment:
This may not need to be returning a final string, after all _feedbackToUser is not a final string